errbot/repo_manager.py

```diff
@@ -106,12 +106,12 @@ def check_for_index_update(self):
             log.info('No repo index, creating it.')
             self[REPO_INDEX] = {LAST_UPDATE: 0}
```

Comment:
I think it might be better to directly call `self.index_update()` here instead of relying on the calculation below. That should also fix the issue you are experiencing.